### DISCUSSION TITLE
[Ultrawide] Street Fighter 6: Turn grey bars into black bars

### DISCUSSION THREAD

**Discussion by codeeno** [praydog/REFramework#858] [General Questions] [answered]
Hey guys,

So for ultrawide users, SF6 has these awful grey bars to the left and right of the screen which isn't great for OLED users. It looks like this:

![2984106813_preview_Screnshot3](https://github.com/praydog/REFramework/assets/60033298/9efc63d3-0b5b-4fbb-8c21-cce59d26ad64)

Personally, I am not a fan of turning off the bars completely (as REFramework does when selecting the UW option) because their absence fundamentally breaks some of the game's core mechanics. I would much rather prefer being able to turn them completely black which makes them less noticeable and is better for my OLED panel. The only fix for now is to manually switch the resolution to 2560x1440 through the OS and/or set up a script which does this switch automatically when you launch the game. Is there a way REFramework could help with this?

**Comment by praydog** [comment, marked as answer]
yes.

**Comment by Mythiiosa** [comment]
Hey, I was just wondering if you were able to figure out how to turn the bars black with REFramework?
If so could you PLEASE share the method.

I can't take these grey bars any longer lol
TY!
